x/oracle/abci.go

```diff
@@ -47,10 +47,10 @@ func CalcPrices(ctx sdk.Context, params types.Params, k keeper.Keeper) error {
 	}
 
 	// voteTargets defines the symbol (ticker) denoms that we require votes on
-	voteTargets := make([]string, 0)
+	voteTargets := make(map[string]bool, 0)
 	voteTargetDenoms := make([]string, 0)
 	for _, v := range params.AcceptList {
-		voteTargets = append(voteTargets, v.SymbolDenom)
+		voteTargets[v.SymbolDenom] = true // unique symbol denoms <Note: we are allowing duplicate symbol denoms>
```

Comment:
It could be programmed in a more strait forward way but I believe it does the checks it needs to such as this spot Sai pointed out. https://github.com/umee-network/umee/blob/89ae44ad465449ce99af16ef301695c0b7e60b60/x/oracle/keeper/msg_server.go#L101-L105. Looks like this update fulfills the intended outcome - removing possible duplicate denoms from voteTargets.